dataactcore/utils/jobQueue.py

```diff
@@ -110,9 +110,13 @@ def generate_e_file(task, submission_id, job_id, interface_holder_class,
             filter(AwardFinancialAssistance.submission_id == submission_id).\
             distinct()
         dunsSet = {r.awardee_or_recipient_uniqu for r in d1.union(d2)}
+        dunsList = list(dunsSet)    # get an order
 
+        rows = []
+        for i in range(0, len(dunsList), 100):
+            rows.extend(fileE.retrieveRows(dunsList[i:i+100]))
```

Comment:
Is this just to retrieve the rows 100 at a time, instead of getting all of them like below (on the deleted line 115)?